go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	cosmossdk.io/errors v1.0.0-beta.7
 	cosmossdk.io/math v1.0.0-beta.4
 	github.com/armon/go-metrics v0.4.1
-	github.com/cosmos/cosmos-sdk v0.46.9
```

Comment:
do we need update [versiondb](https://github.com/crypto-org-chain/cronos/blob/a2f6ff635c92e00fdc9a4ca9e600b3e12bd986cc/versiondb/go.mod#L8) as well

Reply:
yes, done